### PR TITLE
configure prometheus to display stress metrics

### PR DESCRIPTION
**Pull request by larisau**

### CODE REVIEW

File: sdcm/cluster.py [hunk 1]
```diff
@@ -556,9 +556,9 @@ def download_prometheus_data_dir(self):
         self.remoter.receive_files(src=self.prometheus_data_dir, dst=dst)
 
     def _write_prometheus_cfg(self, targets):
-        targets_list = ['%s:9103' % str(ip) for ip in targets]
-        scylla_targets_list = ['%s:9100' % str(ip) for ip in targets]
-        node_exporter_targets_list = ['%s:9180' % str(ip) for ip in targets]
+        scylla_targets_list = ['%s:9100' % str(ip) for ip in targets['db_nodes'] + targets['loaders']]
+        node_exporter_targets_list = ['%s:9180' % str(ip) for ip in targets['db_nodes'] + targets['loaders']]
+        loader_targets_list = ['%s:9103' % str(ip) for ip in targets['loaders']]
```
Comment (amoskong) [inline sdcm/cluster.py]: The variables' name and ports don't match.

- node_exporter: 9100
- prometheus API: 9180
- collectd_exporter: 9103

collectd_exporter was used by old scylla, we won't support that?

File: sdcm/cluster.py [hunk 2]
```diff
@@ -567,23 +567,22 @@ def _write_prometheus_cfg(self, targets):
...
 scrape_configs:
-- job_name: orig-scylla
+- job_name: scylla
   honor_labels: true
   static_configs:
   - targets: %s
 
-scrape_configs:
-- job_name: scylla
+- job_name: node_exporter
```
Comment (amoskong) [inline sdcm/cluster.py]: The job names also need to be fixed.

File: sdcm/cluster.py (hunk 1 above)
Comment (larisau) [reply sdcm/cluster.py]: on loaders only - for stress metrics

Comment (roydahan) [reply sdcm/cluster.py]: No need for collectd on scylla nodes anymore.
Also, prometheus ignore the orig-scylla job.

Comment (amoskong) [reply sdcm/cluster.py]: Got it.

I already approved the PR.

**Comment by amoskong** [comment]
We need this for branch-2.1, right?  @roydahan

**Comment by amoskong** [comment]
Backported the PR to branch-2.1

```
commit 57867cce06a5326106e187f1756a624c20d38565
Author: Larisa Ustalov <larisau@scylladb.com>
Date:   Thu Nov 23 08:24:15 2017 +0200

    fix prometheus port numbers
    (https://github.com/prometheus/prometheus/wiki/Default-port-allocations)
    
    (cherry picked from commit 5173e5acd2883c2a93e8289b8c079f06d478c2ea)

commit 203e02032ec0b6bb150417a8bd7d9cb47abebdb7
Author: Larisa Ustalov <larisau@scylladb.com>
Date:   Wed Nov 22 19:53:37 2017 +0200

    configure prometheus to display stress metrics
    
    (cherry picked from commit f24510fc4407990bda4dee96e1fa02d29cb06cbb)
``

**Pull request merged**